docs/quickstart/standalone.md

```diff
@@ -22,11 +22,19 @@ To get the [Python CLI](../reference/cli.ipynb) and [API](../reference/api/index
 from [PyPI]:
 
 ```bash
-python -m pip install --pre jupyterlite-core
+python -m pip install jupyterlite-core
```

Comment:
maybe we add `sphinx-design` and do the tab thing? or just plain old `<details>`.

not sure if it goes here, or somewhere else, but mentioning the `jupyterlite-core[extras]` and the `jupyterlite-core-with-{extras}` would be good as well.

Reply:
Not sure it should go there. This standalone guide is part of the "Get Started" guides, which should stay simple.

There are links to other the guides at the bottom of the page.

Reply:
sure, that's fine. then i'd stick with `jupyterlite` rather than `-core` so that you can actually _do_ some interactive computing.